gokey_test.go

```diff
@@ -82,6 +83,22 @@ func TestGetPass(t *testing.T) {
 	if (strings.Compare(pass1Example1, pass1Example1Retry) != 0) || (strings.Compare(pass1Example1Seed1, pass1Example1Seed1Retry) != 0) {
 		t.Fatal("passwords with same invocation options do not match")
 	}
+
+	// Testing GOKEY_MASTER environment variable
+	os.Setenv("GOKEY_MASTER", "pass1")
```

Comment:
I think this test is useless, because we're not exercising the code from the cmdline utility at all. This test just goes through the library functions, but env handling is done via cmdline tool.

I think we need a different test here, which invokes `gokey` tool twice:
  * once specifying the password on the cmdline and
  * once specifying the password in env variable
and compare the results

Reply:
If there is a pattern to run sub-process commands within a testing scope or run I suppose that would be one option. I am not aware of others.

Reply:
we can probably add commands directly to https://github.com/cloudflare/gokey/blob/main/.github/workflows/ci.yml (we would need to compile the cmdline tool first anyway)

Reply:
We still haven't resolved this